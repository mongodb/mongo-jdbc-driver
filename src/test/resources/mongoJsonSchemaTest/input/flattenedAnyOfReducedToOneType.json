{
  "bsonType": "object",
  "properties": {
    "bar": {
      "anyOf": [
        {
          "anyOf": [
            {
              "anyOf": [
                {
                  "bsonType": "array",
                  "items" : {"bsonType": "int"},
                  "additionalProperties": true
                }
              ]
            }
          ]
        }
      ]
    }
  }
}
