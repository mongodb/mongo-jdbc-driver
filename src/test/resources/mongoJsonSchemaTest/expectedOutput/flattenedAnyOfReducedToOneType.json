{
  "bsonType": "object",
  "properties": {
    "bar": {
      "bsonType": "array",
      "items": {
        "bsonType": "int"
      },
      "additionalProperties": true
    }
  }
}
